Sources/CAFCastBuilder.swift

```diff
@@ -294,6 +349,9 @@ import GoogleCast
         mediaInfoData["assetReferenceType"] = self.assetReferenceType.description
         mediaInfoData["contextType"] = self.playbackContextType.description
         mediaInfoData["protocol"] = self.httpProtocol.description
+        mediaInfoData["urlType"] = self.urlType.description
+        mediaInfoData["streamerType"] = self.streamerType.description
```

Comment:
what are the defaults?

Reply:
These were set as the defaults:   
 internal var urlType: CAFUrlType = .playmanifest
 internal var streamerType: CAFStreamerType = .applehttp

Reply:
Oh, saw it now in the top of the PR :-)
I'm not sure we need it, these are optional anyway and we should let player decide at least about the streamerType.
For DRM for example this will cause a bug unless app defines DASH.
Same for Android BTW if we set it like this @giladna

Reply:
@OrenMe 
in android there is no default right now in the cast code

i do not send it 

i think the code after Nilit update  is ok